cmd/root.go

```diff
@@ -68,6 +69,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&configFile, "config", "/etc/aurora/australis.yml", "Config file to use.")
 	rootCmd.PersistentFlags().BoolVar(&toJson, "toJSON", false, "Print output in JSON format.")
 	rootCmd.PersistentFlags().StringVarP(&logLevel, "logLevel", "l", "info", "Set logging level ["+internal.GetLoggingLevels()+"].")
+	rootCmd.PersistentFlags().Int64VarP(&timeout, "timeout", "t", 20, "Gorealis timeout (default: 20 seconds)")
```

Comment:
Let's use [DurationVarP](https://godoc.org/github.com/spf13/pflag#DurationVarP) instead of Int64VarP

Reply:
will do.